src/components/LogoSubtitle/LogoSubtitle.jsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { StyledLogoSubtitle } from "./style";
```

Comment:
You don't need to add everywhere LogoSubtitle as it is already local to this component. You can name it just `Wrapper`